anaconda_linter/lint/check_completeness.py

```diff
@@ -44,17 +46,27 @@ def check_recipe_legacy(self, recipe: Recipe) -> None:
 
 class missing_build_number(LintCheck):
     """
-    The recipe is missing a build number
+    missing a build number
```

Comment:
```suggestion
    The recipe is missing a build number
```

Just to match the others. This also reads better when shown to the user.